cosmos.override.js

```diff
@@ -18,9 +18,10 @@ module.exports = (c) => ({
     alias: {
       "react-error-overlay": `${__dirname}/src/panel/util/ErrorOverlay.ts`,
     },
-  },
-  node: {
-    fs: "empty",
+    fallback: {
+      fs: false,
+      path: false,
```

Comment:
Non blocking - any idea what update caused us to need to use `path` in the browser?

Reply:
I am facing this issue when I build cosmos with webpack 5 
![image](https://user-images.githubusercontent.com/12056467/117406795-38adfd00-af16-11eb-8e69-60cb2aa39cf5.png)

I will upgrade electron later and check if this is still needed

Reply:
Interesting - can you try building cosmos but changing this to be `BUILD_ENV=extension` - https://github.com/FormidableLabs/urql-devtools/blob/master/package.json#L25

I can't think of any good reason as to why we would want to use the electron version of the devtools in cosmos (other than me being a pleb and mixing up the two)